test-lib/pom.xml

```diff
@@ -88,4 +88,40 @@
     </plugins>
   </build>
 
+  <profiles>
```

Comment:
Instead of relying on profile, mind if you follow what [integration-test is doing](https://github.com/cloudera/livy/pull/217/files#diff-652e444a472b79b385022ec7dd221790R213)?